packages/article/src/Infrastructure/Sulu/Content/ArticleDataProvider.php

```diff
@@ -182,7 +188,9 @@ protected function resolveFilters(
             $filter['categoryOperator'] = \strtoupper($filters['categoryOperator']);
         }
         if (isset($filters['tags'])) {
-            $filter['tagIds'] = $filters['tags'];
+            /** @var int[] $tags */ // TODO check whats correct here sulu defines string, content bundle int
+            $tags = $filters['tags'];
+            $filter['tagIds'] = $tags;
```

Comment:
need to check this cast whats correct here. may content and sulu is handling tags here differently and so differently typed